plugin/metadata/provider.go

```diff
@@ -48,6 +49,18 @@ type Provider interface {
 // Func is the type of function in the metadata, when called they return the value of the label.
 type Func func() string
 
+// IsLabel check that the provided name looks like a valid label name
+func IsLabel(label string) bool {
+	parts := strings.Split(label, "/")
+	return len(parts) == 2 && len(parts[0]) > 0 && len(parts[1]) > 0
+}
+
+// IsMetadataSet check that the ctx is initialized to receive metadata information
```

Comment:
This is not needed

Reply:
you mean : verifying that a Label has only 2 parts and each is non empty ?

Reply:
for the islabel part, yes.